File: Patterns in Java/Pattern_19.java
import java.util.Scanner;

public class Pattern_19 {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int rows, space;
    char ch = 'A';
    System.out.print("Enter No of Rows: ");
    rows = sc.nextInt();

    for (int i = 1, k = 0; i <= rows; i++, k = 0) {
      for (space = 1; space <= rows - i; space++) {
        System.out.print("  ");
      }
      while (k != 2 * i - 1) {
        System.out.print(ch + " ");
        k++;
      }
      System.out.println();
      ch++;
    }
  }
}

File: Patterns in C/Pattern 19.cpp
# include <stdio.h>
int main()
{
	int rows,space;
	char ch='A';
	printf("Enter No of Rows ");
	scanf("%d",&rows);
	
	for(int i=1, k=0; i<=rows; i++, k=0)
	{
		for(space=1; space<=rows-i; space++)
		{
			printf("  ");
		}
		while(k != 2*i-1)
		{
			printf("%c ",ch);
			k++;
		}
		printf("\n");
		ch++;
	} 
	return 0;
}
